torch/_decomp/decompositions.py

```diff
@@ -630,6 +630,11 @@ def soft_margin_loss_backward(
     return grad_input
 
 
+@register_decomposition(aten.dist)
+def dist(input: Tensor, other: Tensor, p: float = 2):
```

Comment:
This would be better suited for a ref, but fine

Reply:
This was done as part of a brief hackathon, so we were looking for the smallest tasks possible :P